src/autopas/LogicHandler.h

```diff
@@ -181,23 +181,32 @@ class LogicHandler {
   /**
    * @copydoc AutoPas::reserve()
    *
-   * Reserves space in the particle buffers.
+   * Reserves space in the particle buffers and the container. Estimates number of halo particles.
```

Comment:
What is the use of this if you use copydoc?

Reply:
Rewrote this doc: 7fc9847